community/settings.py

```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+from .filters import NoDebugFilter
```

Comment:
add a blank line after system libraries, before local imports.

Reply:
umm... you mean add two blanks between `import os` and `from .filters ...` or a blank